indico/modules/rb_new/blueprints/user.py

```diff
@@ -24,3 +24,4 @@
 
 _bp.add_url_rule('/', 'roombooking', landing.RHRoomBookingLanding)
 _bp.add_url_rule('/<path:path>', 'roombooking', landing.RHRoomBookingLanding)
+_bp.add_url_rule('/rooms', 'available_rooms', landing.RHRoomBookingSearch)
```

Comment:
what about also doing `/api/...` for the JSON endpoints like we did with event logs?